editor/components/icon-button/index.js

```diff
@@ -10,12 +10,13 @@ import './style.scss';
 import Button from '../button';
 import Dashicon from '../dashicon';
 
-function IconButton( { icon, label, className, ...additionalProps } ) {
+function IconButton( { icon, text, label, className, ...additionalProps } ) {
 	const classes = classnames( 'editor-icon-button', className );
 
 	return (
 		<Button { ...additionalProps } aria-label={ label } className={ classes }>
 			<Dashicon icon={ icon } />
+			{ text ? <span>{ text }</span> : null }
```

Comment:
Ah I see :) Though you meant writing `{ text ? <span>{ text }</span> : null }` differently.